### PR TITLE
Rename struct to suit new purpose of adding command abbreviation.

### PR DESCRIPTION
**Pull request by cprince-foreflight**
This also updates the `promptForAction` method to have it use the abbreviation.

I've tested this with a local, changed, version of ffm-build-tools.

My plan is to:
1) Get this merged into your main branch of `swift-utiliities`
2) Re-test my updates to ffm-build-tools using that branch.
3) Add a new tag to your `swift-utiliities`-- will have to bump up the main version as this is a breaking change.

### CODE REVIEW

File: Sources/swift-utilities/CommandLineAction.swift [hunk 1]
```diff
@@ -0,0 +1,21 @@
+//
+//  CommandLineAction.swift
+//  
+//
+//  Created by Bill Gestrich on 12/8/21.
+//
+
+import Foundation
+
```
Comment (bill-foreflight) [inline Sources/swift-utilities/CommandLineAction.swift:9]: To make this all a little more clear, how about something like this:

- invocationName: String
- shortInvocationName: String (if you want to support both full/abbreviated)

That may help to disambiguate the difference between these things and "name". The name is a bit of misnomer -- that was really more of a command description.

Comment (cprince-foreflight) [reply Sources/swift-utilities/CommandLineAction.swift:9]: In ffm-build-tools, the `name` serves two purposes now-- both as the display name for the menu and as a name to be used to invoke the option at the command line. E.g., ./tools.sh "Reset All Caches". `invocationName` doesn't seem to capture the display name usage. How about `longName` and `shortName` given that both are are actually used in display and for invocation now. E.g., this is the way the ./tools.sh menu is about to appear:

```
<Snip-- as this is private>
```

Comment (bill-foreflight) [reply Sources/swift-utilities/CommandLineAction.swift:9]: Changing `name` to `longName` and adding `shortName` works. Thanks

Comment (cprince-foreflight) [reply Sources/swift-utilities/CommandLineAction.swift:9]: Done

**Review by bill-foreflight — APPROVED** [review]

**Pull request merged**